versioned/persist/dynamodb/src/test/java/org/projectnessie/versioned/persist/dynamodb/ITDynamoDatabaseAdapter.java

```diff
@@ -16,7 +16,12 @@
 package org.projectnessie.versioned.persist.dynamodb;
 
 import org.projectnessie.versioned.persist.tests.AbstractDatabaseAdapterTest;
+import org.projectnessie.versioned.persist.tests.extension.NessieDbAdapterConfigItem;
 import org.projectnessie.versioned.persist.tests.extension.NessieExternalDatabase;
 
 @NessieExternalDatabase(LocalDynamoTestConnectionProviderSource.class)
+@NessieDbAdapterConfigItem(name = "retry.initial.sleep.millis.lower", value = "25")
+@NessieDbAdapterConfigItem(name = "retry.initial.sleep.millis.upper", value = "50")
+@NessieDbAdapterConfigItem(name = "retry.max.sleep.millis", value = "150")
+@NessieDbAdapterConfigItem(name = "commit.timeout", value = "500")
```

Comment:
nit: is this intentionally missing the unit? i.e. `.millis`

Reply:
See how those are mapped to actual method on DatabaseAdapterConfig